### ISSUE TITLE
Create placeholder response if the product is present on Open (Beauty/PetFood/Products) Facts

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#497]
#### Web: 
- Add a link to the product on Open (Beauty/PetFood/Products) Facts
- Do not make the placeholder indexable

#### Mobile:
- Send back a product with title: "Present on Open Beauty Facts" and the OBF url as URL on official site?

### Technical decision
- https://docs.google.com/document/d/1WPD23Jf-GMstjiK2gOZGayqECwxtucFqr7ijuyxDN4Q/edit

### Part of
- #5505 
- #5536 
- #5555

**Comment by hangy** [comment]
Right now, this stuff runs on the same server, so some hacks could make this easy, but for long-term support (and possibly to support other future projects), this would need some "interwiki" like feature/API right? This probably also applies #319 and #320.

**Comment by teolemon** [comment]
Yes, long term, we need a decent system. But a hack would be very useful to improve the mobile user experience (at the minimum propose to install Open Beauty Facts on iPhone, or even customize behaviour (on Android))

**Comment by teolemon** [comment]
Just reading the .sto file and returning `cosmetic=true` would be enough for the mobile teams to solve the issue.

**Comment by teolemon** [comment]
This could be a new small new route to avoid breaking anything.

**Comment by teolemon** [comment]
I'm adding the technical decision taken a while ago on universal product API @stephanegigandet @hangy @g123k : https://docs.google.com/document/d/1WPD23Jf-GMstjiK2gOZGayqECwxtucFqr7ijuyxDN4Q/edit
